src/components/Graph/MaxPrice.tsx

```diff
@@ -1,39 +1,24 @@
-import { HistoryData } from '@shapeshiftoss/types'
 import { AxisScale } from '@visx/axis'
 import { Text } from '@visx/text'
+import { ReactNode } from 'react'
 
 export interface LineChartProps {
-  data: HistoryData[]
   xScale: AxisScale<number>
-  yScale: AxisScale<number>
   width: number
   yMax: number
   label: string
   xDate: Date
   yText: number
-  xText: number
-  margin: { top: number; right: number; bottom: number; left: number }
   hideBottomAxis?: boolean
   stroke: string
   hideLeftAxis?: boolean
   top?: number
   left?: number
-  children?: React.ReactNode
+  children?: ReactNode
```

Comment:
`React` is available as a global here and this change isn't related to this rule (tried it out locally with the previous version and I'm having no warnings), could you revert this chunk? 🙏🏽

Reply:
@gomesalexandre I agree - pragmatically I'm going to push this through so as not to let this change require another round of owner reviews.